code/datums/uplink_items/uplink_general.dm

```diff
@@ -74,6 +74,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	var/list/uplinktypes = list() // Empty list means it is in all the uplink types. Otherwise place the uplink type here.
 	var/list/excludefrom = list() // Empty list does nothing. Place the name of uplink type you don't want this item to be available in here.
 	var/list/job = null
+	var/list/species = null
```

Comment:
Please autodoc this (add a comment on the line above that looks like `/// Your comment here`) to explain this new variable